packages/react/src/components/cookieConsent/CookieConsent.stories.tsx

```diff
@@ -722,6 +723,144 @@ export const DebugVersion = (args) => {
+              id: 'preferences1',
+              name: 'Preference 1',
+              hostName: 'Host name',
+              description:
+                'Description lectus lacinia sed. Phasellus purus nisi, imperdiet id volutpat vel, pellentesque in ex. In pretium maximus finibus',
+              expiration: '1h',
+            },
+            {
+              id: 'preferences2',
+              name: 'Preference 2',
+              hostName: 'Host name',
+              description:
+                'Description lectus lacinia sed. Phasellus purus nisi, imperdiet id volutpat vel, pellentesque in ex. In pretium maximus finibus',
+              expiration: '1 years',
+            },
+            {
+              id: 'preferences3',
+              name: 'Preference 3',
+              hostName: 'Host name',
+              description:
+                'Description lectus lacinia sed. Phasellus purus nisi, imperdiet id volutpat vel, pellentesque in ex. In pretium maximus finibus',
+              expiration: '2h',
+            },
+          ],
+        },
+        {
+          commonGroup: 'statistics',
+          cookies: [
+            {
+              commonCookie: 'matomo',
+            },
+            {
+              id: 'someOtherConsent',
+              name: 'Other consent',
+              hostName: 'Host name',
+              description:
+                'Description lectus lacinia sed. Phasellus purus nisi, imperdiet id volutpat vel, pellentesque in ex. In pretium maximus finibus',
+              expiration: '1h',
+            },
+          ],
+        },
+      ],
+    },
+
+    language: {
+      onLanguageChange,
+    },
+    onAllConsentsGiven: (consents) => {
+      if (consents.matomo) {
+        //  start tracking
+        // window._paq.push(['setConsentGiven']);
+        // window._paq.push(['setCookieConsentGiven']);
+      }
+    },
+    onConsentsParsed: (consents, hasUserHandledAllConsents) => {
+      if (consents.matomo === undefined) {
+        // tell matomo to wait for consent:
+        // window._paq.push(['requireConsent']);
+        // window._paq.push(['requireCookieConsent']);
+      } else if (consents.matomo === false) {
+        // tell matomo to forget conset
+        // window._paq.push(['forgetConsentGiven']);
+      }
+      if (hasUserHandledAllConsents) {
+        // cookie consent modal will not be shown
+      }
+    },
+    focusTargetSelector: '#focused-element-after-cookie-consent-closed',
+  };
+
+  const Application = () => {
+    return (
+      <div>
+        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+        <h1 id={contentSource.focusTargetSelector?.replace('#', '')} tabIndex={0}>
+          This is an example application with cookie consent modal in a Portal
+        </h1>
+        <p>Modal is rendered to the given element.</p>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <PortalModal contentSource={contentSource} rootId="cookie-modal-portal-container" />
```

Comment:
I guess it's a good to have the optional prop `rootId`, but try to emphasise users that the modal should be rendered to the first element of the body.